### PR TITLE
Python env

### PR DESCRIPTION
**Pull request by PythonGurl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .DS_Store
-env/
\ No newline at end of file
+env/
+*.pyc
+.env
```
Comment (nerner-code) [inline .gitignore:4]: Please add a newline to the end of this file.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from dash import Dash, dcc, html, Input, Output
+import os
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = Dash(__name__, external_stylesheets=external_stylesheets)
+
+server = app.server
```
Comment (nerner-code) [inline app.py:8]: Do we use `server` anywhere?  I don't see any usages.  If it isn't used, can we delete it?

File: app.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from dash import Dash, dcc, html, Input, Output
+import os
```
Comment (nerner-code) [inline app.py:2]: I don't think we use the `os` module.  Lets delete this import.

**Review by nerner-code — COMMENTED** [review]
Can you add a screenshot of the website to this PR as well?

**Comment by nerner-code** [comment]
Can you please add a screenshot of the website to this PR?

**Comment by PythonGurl** [comment]
Boom!
image

**Review by nerner-code — APPROVED** [review]

**Pull request merged**